wrappers/awskms/awskms.go

```diff
@@ -97,6 +100,16 @@ func (k *Wrapper) SetConfig(_ context.Context, opt ...wrapping.Option) (*wrappin
 		return nil, fmt.Errorf("key id not found in env or config for aws kms wrapper configuration")
 	}
 
+	k.currentKeyId.Store(k.keyId)
```

Comment:
Should this not be wrapped in an if k.keyId != nil ? I believe the Store will panic if k.keyNotRequired is set?

Reply:
nvm keyId will be a blank not a nil...